services/net/nlp/TNO.Services.NLP.csproj

```diff
@@ -23,6 +23,10 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.ML" Version="4.0.0" />
     <PackageReference Include="OpenNLP.NET" Version="1.9.4.1" />
+    <PackageReference Include="System.Formats.Asn1" Version="8.0.1" />
+    <PackageReference Include="System.Security.Cryptography.Pkcs" Version="8.0.1" />
+    <PackageReference Include="System.Text.Json" Version="8.0.5" />
+    <PackageReference Include="TNO.Services" Version="1.0.0" />
```

Comment:
same question as above, why use version 1.0.0?

Reply:
A project reference will use the latest code changes, and that would result in .NET9.0.